app/services/summary_service.rb

```diff
@@ -22,12 +24,16 @@ def self.summary(summary_type, prison)
     buckets = { allocated: Bucket.new(sortable_fields),
                 unallocated: Bucket.new(sortable_fields),
                 pending: Bucket.new(sortable_fields),
-                new_arrivals: Bucket.new(sortable_fields)
+                new_arrivals: Bucket.new(sortable_fields),
+                handovers: Bucket.new(sortable_fields)
     }
 
     offenders = prison.offenders
     active_allocations_hash = AllocationService.active_allocations(offenders.map(&:offender_no), prison.code)
 
+    # Add POM & COM allocation details to offenders. We use this information in 'allocated' and 'handovers' views.
```

Comment:
I'd be happy to drop this comment, the method name is good enough to express it, I think.